app/views/ansible_roles/index.html.erb

```diff
@@ -1,47 +1,19 @@
+<%= webpacked_plugins_js_for :foreman_ansible %>
+<%= webpacked_plugins_css_for :foreman_ansible %>
```

Comment:
`webpacked_plugins_css_for` is deprecated, plugin css is already loaded.'

Reply:
I think it would be best if #715 could be merged, so I can rebase this branch.

Reply:
That has been merged.